docs/content/en/docs/operator-manual/piped/adding-a-cloud-provider.md

```diff
@@ -69,8 +69,8 @@ spec:
       type: CLOUDRUN
       config:
         project: gcp-project
```

Comment:
This too.

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/2078/commits/a251c4e123c5a21841ab337c1fe8f1d9a0c3ad19 🙏